src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -90,11 +125,27 @@ const SampleChatScreen = () => {
         bounces={false}
         onLayout={scrollToEnd}
         keyExtractor={(item, index) => item?.id || index.toString()}
+        ListFooterComponent={
+          ownerChat ? (
+            <BaseSystemChat
+              componentType="SINGLE"
+              messageSingle={`You’re anonymously messaging ${selectedChannel?.name}.\nThey are still able to block you`}
+            />
+          ) : null
+        }
         renderItem={renderChatItem}
       />
       <View style={styles.inputContainer}>
-        <AnonymousInputMessage onSendButtonClicked={sendChat} type={ANONYMOUS} />
+        <InputMessageV2
+          onSendButtonClicked={sendChat}
+          type={ANONYMOUS}
+          emojiCode={selectedChannel?.rawJson.channel.anon_user_info_emoji_code}
+          emojiColor={selectedChannel?.rawJson.channel.anon_user_info_color_code}
+          username={selectedChannel?.name}
+          onToggleConfirm={moveChatSigned}
+        />
       </View>
+      <Loading visible={loading} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `ListFooterComponent` and replaces `AnonymousInputMessage` with `InputMessageV2`. These changes seem to be part of the feature being implemented and don't appear to introduce any issues. The loading state is also handled well with the `Loading` component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->